parlor_test.go

```diff
@@ -3,48 +3,148 @@ package parlor_test
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/blocky/parlor"
-	"github.com/blocky/parlor/internal/mocks"
 )
 
-type ParlorTestSuite struct {
+type CallCounter struct {
+	Test    int
+	Subtest int
+}
+
+func (c CallCounter) Assert(t *testing.T, test, subtest int) {
+	assert.Equal(t, test, c.Test)
+	assert.Equal(t, subtest, c.Subtest)
+
+}
+
+// this test makes sure everything compiles and total call cound it correct
+// if we just have a setup
```

Comment:
"call cound it correct" -> "call count is correct"

Reply:
done